### DISCUSSION TITLE
Feature Request: Add option to hide right panel with "People you may know" and Ads

### DISCUSSION THREAD

**Discussion by woodynco** [jordwest/news-feed-eradicator#373] [Ideas]
This is the last bit of distracting and useless info on FB newsfeed.
Top part with event invates and birthdays is good, but "People you may know" and Right panel Ads are distracting.

Would be great to block that specific block.

**Comment by Zabari** [comment]
Is this something this project is interested in tackling @jordwest ? If so, I'd like to take a look at it.

**Comment by stephenpcook** [comment]
I don't get the "people you may know" (only "n event invitations").  I'd expect that removing the sponsored adverts would more likely be the role of an ad-blocker instead. Just my opinion
